guides/v2.1/ui_comp_guide/concepts/ui_comp_linking_concept.md

```diff
@@ -157,15 +159,17 @@ For example of `listens` usage in Magento code see [`new_category_form.xml`, lin
 
 ## Template strings usage {#string_templ}
 
-The `value` options of linking properties can contain template strings in the `'${...}'` format. During component’s initialization, values in this format are processed as template strings using [ES6 templates](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Template_literals). In browsers that do not support ES6 templates, these values are processed as underscore templates.
+The options of linking properties can contain template strings in the `'${...}'` format. During the component’s initialization, values in this format are processed as template strings using [ES6 templates](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Template_literals). In browsers that do not support ES6 templates, these values are processed as underscore templates.
 
 So if we put a variable name in `'${...}'`, it is processed into a string representation of the variable’s value.
 
 When working with UI components, we often need to use the string representation of a certain property of the {% glossarytooltip 9bcc648c-bd08-4feb-906d-1e24c4f2f422 %}UI component{% endglossarytooltip %}. To address a property of the UI component in the scope of this component, the `$.someProperty` syntax is used.
 
 As a result, if the component's property is the variable for the template string, we get notation similar to the following:
 
-    '${ $.provider }' 
+    '${ $.provider }:foo' 
+    
+If the string would be built at runtime it would be equivalent to `this.provider + '.foo'`.
```

Comment:
It would be equal to `this.provider + ':foo'`

Reply:
Right, fixed.